tests/unit/utils/test_config.py

```diff
@@ -27,11 +38,19 @@
             ),
         ),
         (
+            # Given: single model, one variable
+            # Expected: Config object with one model, no variable
             """
-            models:
-              - name: single_model
-                deployment_id: single_dep
-                temperature: 1
+            {
+                "VARIABLE_NAME": "value",
+                "models": [
+                    {
+                        "name": "single_model",
+                        "deployment_id": "single_dep",
+                        "temperature": 1
+                    }
+                ]
+            }
             """,
             Config(
```

Comment:
will add the `"VARIABLE_NAME": "value"` expected config too?

Reply:
added unit test for the env variables